IFC4x3/Templates/Product Shape/Product Geometric Representation/Body Geometry/Body AdvancedSweptSolid Geometry/Body SectionedSolidHorizontal/DocTemplateDefinition.xml

```diff
@@ -16,18 +16,7 @@
 										</DocModelRuleAttribute>
 										<DocModelRuleAttribute CardinalityMin="1" CardinalityMax="1" Name="RepresentationIdentifier" Identification="Identifier">
 											<Rules>
-												<DocModelRuleEntity id="IfcLabel_2nTsBx4yb0wfI4IH_ymbNg" Name="IfcLabel" UniqueId="b17762fb-13c9-40ea-9484-491fbcc255ea">
-													<Rules>
-														<DocModelRuleConstraint>
-															<Expression xsi:type="DocOpStatement" Operation="compareequal">
-																<Reference xsi:type="DocOpReference">
-																	<EntityRule xsi:type="DocModelRuleEntity" xsi:nil="true" href="IfcLabel_2nTsBx4yb0wfI4IH_ymbNg" />
-																</Reference>
-																<Value xsi:type="DocOpLiteral" Operation="loadstring" Literal="Body" />
-															</Expression>
-														</DocModelRuleConstraint>
-													</Rules>
-												</DocModelRuleEntity>
+												<DocModelRuleEntity Name="IfcLabel" xsi:nil="true" href="IfcLabel_2fgI0VXve2tV6eMzYUyfa9"/>
```

Comment:
This seems odd: no `RepresentationIdentifier=Body` for _SectionedSolidHorizontal_, although specified to have one in the documentation?

Reply:
?? 
![image](https://user-images.githubusercontent.com/25506404/141955080-5bee2131-d3d4-473d-a10b-031e24ec61a2.png)

Reply:
@pjanck this a reference to the rule in the parent template so when you update the parent it will hopefully now filter down its a recent feature in IFC doc. As lars shows when you open it, it displays correctly in IfcDoc